src/foundry/common/utils/helpers.mjs.d.ts

```diff
@@ -219,11 +238,13 @@ export declare function getRoute(
 ): string;
 
 /**
- * Learn the named type of a token - extending the functionality of typeof to recognize some core Object types
- * @param token - Some passed token
+ * Learn the underlying data type of some variable. Supported identifiable types include:
+ * undefined, null, number, string, boolean, function, Array, Set, Map, Promise, Error,
+ * HTMLElement (client side only), Object (catchall for other object types)
+ * @param variable - A provided variable
  * @returns The named type of the token
  */
-export declare function getType(token: unknown): string;
+export function getType(variable: unknown): string;
```

Comment:
I think we could specify a union type of all possible values for the return type.

Reply:
Yea, I misread the implementation as digging into the prototype tree. Late night work before chasing a rabbit.